src/seedu/addressbook/Main.java

```diff
@@ -108,7 +108,7 @@ private void recordResult(CommandResult result) {
     private CommandResult executeCommand(Command command)  {
         try {
             command.setData(addressBook, lastShownList);
-            CommandResult result = command.execute();
+            CommandResult result = command.execute(); // command Result contains a String which is feedback to the user and a list of relevant persons
```

Comment:
Is this comment relevant here?
Should it be in the `@return` JavaDocs for the `execute()` method instead?